src/styles/main.scss

```diff
@@ -1,3 +1,24 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
```suggestion
```
this properties is reason that content not centered by page
![Screenshot from 2023-04-27 13-17-30](https://user-images.githubusercontent.com/33029864/234833929-a3318845-9fd3-4ff8-974a-b28a655c69eb.png)